scala-package/examples/src/main/scala/org/apache/mxnetexamples/neuralstyle/NeuralStyle.scala

```diff
@@ -39,7 +39,7 @@ object NeuralStyle {
   private val logger = LoggerFactory.getLogger(classOf[NeuralStyle])
 
   def preprocessContentImage(path: String, longEdge: Int, ctx: Context): NDArray = {
-    val img = Image(new File(path))
+    val img = Image.fromFile(new File(path))
```

Comment:
Why change this?

Reply:
There was a warning against using Image(). I don't remember what the warning was, though.